### ISSUE TITLE
zc & aoe don't seem to work together

### ISSUE THREAD

**Issue by leviwheatcroft** [splittingred/phpThumbOf#55]
I can get each of these options working alone, but in conjunction they don't work.

The phpthumb readme says zc overrides iar & far, but not aoe.

**Comment by leviwheatcroft** [comment]
Actually maybe this is an upstream problem. See line 1412 of phpthumb.class.php. (I have no idea how to fix it either!)

**Comment by jpdevries** [comment]
I'd suggest trying pthumb or phpthumbof instead. phpthumbof is now over at https://github.com/modxcms/phpthumbof but even there is depreciated.

**Comment by leviwheatcroft** [comment]
Oh I see. Is it officially deprecated? It's just that there's no mention of pthumb in the phpthumbof docs or repositories.

Just I've spent hours trying to find a work around for this problem, in all my searching I never ran across pthumb (which solves the problem nicely).

**Comment by jpdevries** [comment]
Yes it is officially depreciated. We'll update the README. There was a blocker in removing the Extra completely from the MODX Extra site, just can't remember what it was.
